README.zh.md

```diff
@@ -0,0 +1,228 @@
+<img alt="有状态函数" src="docs/fig/stateful_functions_logo.png" width=350px/>
+
+有状态函数是一个[Apache Flink](https://flink.apache.org/)库， __可简化构建分布式有状态应用程序的过程__ 。它基于有着可持久化状态的函数，这些函数可以在强大的一致性保证下进行动态交互。 
+
+有状态函数使强大的状态管理和组合，与AWS Lambda之类的FaaS实现和Kubernetes之类的现代资源编排框架的弹性，快速缩放/零缩放和滚动升级功能相结合成为可能。通过这些特性，它解决了当今许多FaaS设置中[最常被引用的两个缺点](https://www2.eecs.berkeley.edu/Pubs/TechRpts/2019/EECS-2019-3.pdf) ：状态一致和函数之间的高效消息传递。 
+
+本自述文件旨在简要介绍核心概念以及如何进行设置
+以使您开始使用有状态函数。 
```

Comment:
I agree with the translations, but I think it is best preserve line breaks; so that when the corresponding English docs changes, the relevant line in Chinese docs can be quickly located. This also assist with machine learning translations, as a training set for translations can be easily extracted.